packages/rke2-coredns/generated-changes/overlay/templates/daemonset-nodelocal.yaml

```diff
@@ -61,15 +72,19 @@ spec:
           protocol: TCP
         livenessProbe:
           httpGet:
+{{- if not .Values.nodelocal.use_cilium_lrp }}
             host: {{.Values.nodelocal.ip_address}}
+{{- end}}
             path: /health
             port: 8080
           initialDelaySeconds: 60
           timeoutSeconds: 5
         volumeMounts:
+{{- if .Values.nodelocal.use_cilium_lrp }}
         - mountPath: /run/xtables.lock
           name: xtables-lock
           readOnly: false
+{{- end}}
```

Comment:
By this change you are removing the `xtables.lock` when not using cilium_lrp, is that intentional? You'll be changing the current behaviour

Reply:
good catch, it should be the opposite